content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,85 @@
+* In-tree provisioner to CSI driver migration. This applies to a number of in-tree plugins, including [vSphere](https://github.com/kubernetes/enhancements/issues/1491) and [Azure File](https://github.com/kubernetes-sigs/azurefile-csi-driver). Refer to the [In-tree Storage Plugin to CSI Migration Design Doc](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/csi-migration.md#background-and-motivations) for more information. 
+* [Storage capacity tracking for pod scheduling](https://github.com/kubernetes/enhancements/issues/1472): The CSIStorageCapacity API supports exposing currently available storage capacity via CSIStorageCapacity objects and enhances scheduling of pods that use CSI volumes with late binding. The CSIStorageCapacity API is stable as of 1.24. The v1beta1 CSIStorageCapacity API is deprecated in 1.24. Refer to the [Storage Capacity Constraints for Pod Scheduling KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1472-storage-capacity-tracking) for more information. 
+
+### A note about Dockershim
+
+It's safe to say that the removal receiving the most attention with the release of Kubernetes 1.24 is Dockershim. Dockershim was deprecated in 1.20. As noted in the [Kubernetes 1.20 changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#deprecation): "Docker support in the kubelet is now deprecated and will be removed in a future release. The kubelet uses a module called "dockershim" which implements CRI support for Docker and it has seen maintenance issues in the Kubernetes community." With the upcoming release of Kubernetes 1.24, that day is finally here. 
+
+In the article, [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/), the authors succinctly captured the change's impact and encouraged users to remain calm: "Docker as an underlying runtime is being deprecated in favor of runtimes that use the Container Runtime Interface (CRI) created for Kubernetes. Docker-produced images will continue to work in your cluster with all runtimes, as they always have." Several guides have been created with helpful information about migrating from dockershim to other container runtimes. You can find them on the [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) page in the Kubernetes documentation.
+
+For more information about why Kubernetes is moving away from dockershim, check out the aptly named: [Kubernetes is Moving on From Dockershim](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/) and the [updated dockershim removal FAQ](/blog/2022/02/17/dockershim-faq/).
+
+Take a look at the [What you need to do to have a working cluster for 1.24](https://github.com/kubernetes/website/issues/30917) to learn what you need to do to go from a working 1.23 cluster to a working 1.24 cluster. 
```

Comment:
- I'd mention this much earlier in the article, or have an early mention and move this near to the end.
- The link to `https://github.com/kubernetes/website/issues/30917` should instead be a link to `/blog/2022/03/31/ready-for-dockershim-removal/`